tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -35,10 +35,10 @@ def executor_kwargs(test_type, test_environment, run_info_data, **kwargs):
         executor_kwargs["reftest_screenshot"] = kwargs["reftest_screenshot"]
 
     if test_type == "wdspec":
-        executor_kwargs["binary"] = kwargs.get("binary")
-        executor_kwargs["binary_args"] = kwargs.get("args")
-        executor_kwargs["webdriver_binary"] = kwargs.get("webdriver_binary")
-        executor_kwargs["webdriver_args"] = kwargs.get("webdriver_args")
+        executor_kwargs["binary"] = kwargs["binary"]
+        executor_kwargs["binary_args"] = kwargs["binary_args"][:]
```

Comment:
So what's the benefit for using `[:]` instead of just `.copy()`? The latter seems to be in use way more often in this repository.

Reply:
Mostly that `.copy()` on lists is only 3.3+ so I didn't know about it (`.copy()` on dictionaries is much older). I've changed it now.